Runner.java

```diff
@@ -0,0 +1,71 @@
+import by.epam.lab.Purchase;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Runner {
+    public static void main(String[] args) {
+
+        try(Scanner sc = new Scanner(new FileReader("src/in.txt"))){
+            sc.useLocale(Locale.ENGLISH);
+            sc.useDelimiter(System.getProperty("line.separator"));
+
+            // creating an array
+            int purchasesNumber = Integer.parseInt(sc.nextLine());
+            Purchase[] purchases = new Purchase[purchasesNumber];
+
+            // initialise an array
+            String[][] var = new String[purchasesNumber][3];
+            for (int i = 0; i < purchasesNumber; i++) {
+                var[i] = sc.nextLine().split(" ");
+                purchases[i] = new Purchase(Integer.parseInt(var[i][0]), Integer.parseInt(var[i][1]), Purchase.WeekDay.values()[Integer.parseInt(var[i][2])]);
+            }
+
+            // output the array, calculating the mean cost of all purchases, the total cost of al purchases on Monday, the day with whe max purchase cost
+            Purchase maxPurchase = new Purchase();
+            int meanCost = 0;
+            int costMonday = 0;
+            System.out.println(String.format("%s;%d.%02d", purchases[0].getPRODUCT_NAME(), purchases[0].getPRICE()/100, purchases[0].getPRICE()%100));
+            for (Purchase purchase : purchases){
+                if (purchase.getCost() > maxPurchase.getCost()){
+                    maxPurchase = purchase;
+                }
+                if (purchase.getWeekDay() == Purchase.WeekDay.MONDAY){
+                    costMonday += purchase.getCost();
+                }
+                System.out.println(purchase.toString());
+                meanCost += purchase.getCost();
+            }
+            System.out.print("Day with maximum purchase cost : ");

```

Comment:
You should use formatted strings here and (almost) everywhere. Intead of println(String.format()) use printf()